Dockerfile.ubi

```diff
@@ -0,0 +1,55 @@
+# Build the operator binary
+FROM --platform=$TARGETPLATFORM golang:1.17.8 as builder
```

Comment:
Since you opened the PR, we updated the goland version. Could you rebase main?
```suggestion
FROM --platform=$TARGETPLATFORM golang:1.18.1 as builder
```